site/docs/v1/tech/apis/_webauthn-register-start-request-body.adoc

```diff
@@ -5,7 +5,14 @@
 The primary display name for the new passkey.
 
 [field]#name# [type]#[String]# [optional]#Optional# [default]#defaults to **[see description]**#::
-The optional name for the passkey. This is meant to disambiguate passkeys with the same [field]#displayName#. When this parameter is omitted, it will be defaulted to a random four character alphabetic string that is unique among a user's passkeys.
+The optional name for the passkey. This is meant to disambiguate passkeys with the same [field]#displayName#. When this parameter is omitted, it will be defaulted to the user's email address. If the user does not have an email address, this parameter will be defaulted to their username instead.
++
+Prior to version `1.42.0` [field]#name# was defaulted to a random four character alphanumeric string.
+
+[NOTE.note]
+====
+The passkey [field]#name# is set at the time the passkey is registered. Prompts displayed by the WebAuthn JavaScript API will continue to display the original value at the time the passkey was registered even if the user updates their email address or username.
```

Comment:
Is this because the `name` is embedded in the credential? There's no way to change that without re-registration of the passkey, correct? Might want to note that.